examples/issues.yml

```diff
@@ -0,0 +1,27 @@
+---
+# This example file will enable actions that trigger on created or modified GitHub issues. 
+#
+# Note the @maain in `uses:` on the last line. This will call the latest version of the workflow from the `main` branch in the RMI-PACTA/actions repo.
+# You can also specify a tag from that repo, or a commit SHA to pin action versions.
+on:
+  issues:
+    types:
+      [opened, edited, deleted, closed, reopened, labeled, unlabeled, assigned]
+    issue_comment:
+      types: [created, edited, deleted]
+
+name: GH issues
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  issues:
+    name: Run issues workflows
+    uses: RMI-PACTA/actions/.github/workflows/issues@main
+    with:
+      ado_area_path: "2DegreesInvesting\\Test Team 1"
+    secrets:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
In conjunction with setting the `permissions` above, you shouldn't need to pass the `GITHUB_TOKEN` explicitly.